src/korean.js

```diff
@@ -40,108 +72,68 @@ class Korean {
           newSyllable = combineSymbols(newSyllable);
-          if(wordLength <= 2) {
+          if (wordLength <= 2) {
             return newSyllable;
-          } else {
-            return stemWord + newSyllable;
           }
-       case 20:
-        // vowel ㅣ replace with ㅕ(6)
-          newSyllable.push(6)
+          return stemWord + newSyllable;
+        case 20:
+          // vowel ㅣ replace with ㅕ(6)
+          newSyllable.push(6);
           newSyllable = combineSymbols(newSyllable);
-          if(wordLength <= 2) {
+          if (wordLength <= 2) {
             return newSyllable;
-          } else {
-            return stemWord + newSyllable;
           }
+          return stemWord + newSyllable;
         default:
-            // if consonant
-            conjugate = word.slice(0, wordLength - 1);
-            return conjugate.concat('어');
+          // if consonant
+          conjugate = word.slice(0, wordLength - 1);
+          return conjugate.concat('어');
       }
     } // end of first else
   } // end of presentWord function
 
-  doPast(word) {
-    let presentWord = doPresent(word);
+  doPast (word) {
+    console.log(word);
     // stuff for past tense
   }
 
   doFuture (word) {
-    let stem = breakdown(word.slice(0,-1));
+    const stem = breakdown(word.slice(0, -1));
     if (stem[-1] !== 8) {
       stem.push(8);
     }
-    return combineSymbols(stem) + '꺼야';
+    return `${combineSymbols(stem)}꺼야`;
   }
 } // end for class
 
-
-/** the conjugate returns a conjugate verb, but it should be checked for tense, past tense will take present tense verb and then add something to the the verb, and then add an additional letter after **/
-
-/** Additional Functions **/
-// reference: http://www.programminginkorean.com/programming/hangul-in-unicode/composing-syllables-in-unicode/
-// input: hangul character
-// output: returns an array of components that make up the given hangul
-// ex: breakdown('린') outputs [5,20,4]
-const breakdown = (input) => {
-    let total = parseInt(input.charCodeAt(0).toString(16), 16);
-
-    let letterArray = [];
-    total = total - 44032;
-    let initialValue = Math.floor(total / 588);
-    letterArray.push(initialValue);
-    total = total - (588 * initialValue);
-    let medialValue = Math.floor(total / 28);
-    letterArray.push(medialValue);
-    let finalValue = total - (28 * medialValue);
-    if (finalValue > 0) {
-        letterArray.push(finalValue);
-    }
-    return letterArray;
-}
-
-// input: array of 2 or 3 numbers representing modern jamo (components) that make up a hangul syllable
-// output: a hangul character
-const combineSymbols = (input) => {
-    let initialValue = input[0] * 588;
-    let medialValue = input[1] * 28;
-    let finalValue = input[2] ? input[2] : 0;
-    let total = initialValue + medialValue + finalValue + 44032;
-    let finalWord = String.fromCharCode(total);
-    return finalWord;
-}
-
-
-export { Korean };
+export default { Korean };
```

Comment:
Same as for French,
default won't work here. Korean is a class, not a constructor.